src/main/java/com/monumental/controllers/SuggestionController.java

```diff
@@ -0,0 +1,302 @@
+        createSuggestion.setIsRejected(true);
+        return this.createSuggestionRepository.save(createSuggestion);
+    }
+
+    /**
+     * Reject an UpdateMonumentSuggestion with the specified ID, if it exists
+     * @param id - ID of the UpdateMonumentSuggestion to reject
+     * @return UpdateMonumentSuggestion - UpdateMonumentSuggestion object that was rejected
+     * @throws ResourceNotFoundException - If an UpdateMonumentSuggestion with the specified ID does not exist
+     */
+    @PutMapping("/api/suggestion/update/{id}/reject")
+    @PreAuthorize(Authorization.isResearcherOrAbove)
+    public UpdateMonumentSuggestion rejectUpdateSuggestion(@PathVariable("id") Integer id) throws ResourceNotFoundException {
+        UpdateMonumentSuggestion updateSuggestion = this.findUpdateSuggestion(id);
+        updateSuggestion.setIsRejected(true);
+        return this.updateSuggestionRepository.save(updateSuggestion);
+    }
+
+    /**
+     * Reject a BulkCreateMonumentSuggestion with the specified ID, if it exists
+     * @param id - ID of the BulkCreateMonumentSuggestion to reject
+     * @return BulkCreateMonumentSuggestion - BulkCreateMonumentSuggestion object that was rejected
+     * @throws ResourceNotFoundException - If a BulkCreateMonumentSuggestion with the specified ID does not exist
+     */
+    @PutMapping("/api/suggestion/bulk-create/{id}/reject")
+    @PreAuthorize(Authorization.isResearcherOrAbove)
+    public BulkCreateMonumentSuggestion rejectBulkCreateSuggestion(@PathVariable("id") Integer id)
+            throws ResourceNotFoundException {
+        BulkCreateMonumentSuggestion bulkCreateSuggestion = this.findBulkCreateSuggestion(id);
+
+        bulkCreateSuggestion.setIsRejected(true);
+        this.createSuggestionRepository.saveAll(bulkCreateSuggestion.getCreateSuggestions());
+        bulkCreateSuggestion = this.bulkCreateSuggestionRepository.save(bulkCreateSuggestion);
+
+        // Remove images from temporary S3 folder
+        for (CreateMonumentSuggestion createSuggestion : bulkCreateSuggestion.getCreateSuggestions()) {
+            if (createSuggestion.getImages() != null) {
+                for (String imageUrl : createSuggestion.getImages()) {
+                    this.awsS3Service.deleteObject(AwsS3Service.getObjectKey(imageUrl, true));
+                }
+            }
+        }
+
+        return bulkCreateSuggestion;
+    }
+
+    /**
+     * Helper function to find a CreateMonumentSuggestion object with the specified ID, if it exists
+     * @param id - ID of the CreateMonumentSuggestion object to find
+     * @return CreateMonumentSuggestion - CreateMonumentSuggestion object with the specified ID, if it exists
+     * @throws ResourceNotFoundException - If a CreateMonumentSuggestion does not exist with the specified ID
+     */
+    private CreateMonumentSuggestion findCreateSuggestion(Integer id) throws ResourceNotFoundException {
```

Comment:
I guess they could be. They're just a wrapper around calling the JPA Repositories though. I just made them into functions because I found myself re-writing it in every endpoint.